enhancements/windows-containers/monitoring-windows-nodes.md

```diff
@@ -0,0 +1,144 @@
+Hence, we need to ensure that Prometheus operator watches over the Windows
+exporter external service.
+
+### Implementation Details/Notes/Constraints [optional]
+
+To enable monitoring WMCO is be responsible for the following:
+
+* Adding Windows exporter binary to WMCO payload.
+* Installing and managing Windows exporter on the Windows nodes and ensuring 
+  that it runs as a Windows service. 
+* Adding `openshift.io/cluster-monitoring=true` label so that cluster monitoring
+  stack will pick up the service monitor created by WMCO.
+* Adding privileges to create services, endpoint, service monitors in the 
+  `openshift-windows-machine-config-operator` namespace. 
+* Creating a service and endpoints in `openshift-windows-machine-config-operator`
+  namespace that point to Windows exporter endpoint. WMCO will be using default
+  values to define metrics endpoint on Windows exporter which is equivalent to
+  `<internal-ip>:9182/metrics` for a given Windows node. The Endpoints object
+  created in the namespace will consist of subsets of endpoints from all the
+  Windows nodes.
+* Creating a service monitor in `openshift-windows-machine-config-operator` 
+  namespace for service created above.
+  
+To display node and workload graphs WMCO is responsible for the following:
+
+* Adding custom Prometheus rules in `openshift-windows-machine-config-operator` 
+  namespace. The custom recording rules will be created using Windows metrics
+  exposed by the Windows exporter and will have the same names as Linux
+  recording rules. This is to make use of same console queries as Linux.
+* Note that WMCO is unable to display workload graphs for the Windows Nodes. 
+  See [Risks and Mitigations](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/monitoring-windows-nodes.md#risks-and-mitigations) 
+  for details.
+  
+### Risks and Mitigations
+
+The main risk with this proposal is renaming Windows metrics to display workload
+graphs. The pod metrics for linux come from cAdvisor. However, we do not get
+same metrics from cAdvisor for Windows nodes. This becomes a hindrance to 
+display pod graphs by creating custom recording rules to use same console 
+queries as Linux workloads.
+
+Mitigation
+* Use metrics exposed by the Windows exporter to display pod graphs. This also 
+  requires changes in console queries that support OS specific metrics. 
+
+## Design Details
+
+### Test Plan
+
+We plan to add e2e tests to ensure
+* Service, endpoints and Service Monitors are created as expected.
+* Prometheus is able to collect data from Windows exporter.
+* Custom Prometheus rules return Windows data. 
+
+### Graduation Criteria
+
+This enhancement will start as GA
+
+### Upgrade / Downgrade Strategy
+
+* WMCO is responsible for upgrading [Windows exporter](https://github.com/prometheus-community/windows_exporter/tags) 
```

Comment:
Please call out downgrade strategy or the lack of one as we cannot downgrade OLM operators.